cpp/ql/lib/experimental/quantum/OpenSSL/AlgorithmValueConsumers/HashAlgorithmValueConsumer.qll

```diff
@@ -43,8 +71,11 @@ class EVPDigestAlgorithmValueConsumer extends HashAlgorithmValueConsumer {
         ] and
       valueArgNode.asExpr() = this.(Call).getArgument(0)
       or
-      this.(Call).getTarget().getName() = "EVP_MD_fetch" and
+      this.(Call).getTarget().getName() in ["EVP_MD_fetch"] and
       valueArgNode.asExpr() = this.(Call).getArgument(1)
+      or
+      this.(Call).getTarget().getName() = "EVP_DigestSignInit_ex" and
```

Comment:
Consider making the class `instanceof Call` and saying `super.getTarget().hasName("EVP_DigestSignInit_ex")`